README.md

```diff
@@ -223,26 +224,39 @@ The following special character strings are supported:
 > combinations as different operating systems function differently in this
 > regard.
 
-An example of an usage with a selection range:
+Examples of usage with a selection range and `{space}` on button:
 
 ```jsx
-import React from 'react'
-import {render, screen} from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
-
-test('delete characters within the selectedRange', () => {
-  render(
-    <div>
-      <label htmlFor="my-input">Example:</label>
-      <input id="my-input" type="text" value="This is a bad example" />
-    </div>,
-  )
-  const input = screen.getByLabelText(/example/i)
-  input.setSelectionRange(10, 13)
-  userEvent.type(input, '{backspace}good')
-
-  expect(input).toHaveValue('This is a good example')
-})
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("Special characters examples", () => {
+  test("delete characters within the selectedRange", () => {
+    render(
+      <div>
+        <label htmlFor="my-input">Example:</label>
+        <input id="my-input" type="text" value="This is a bad example" />
+      </div>
+    );
+    const input = screen.getByLabelText(/example/i);
+    input.setSelectionRange(10, 13);
+    userEvent.type(input, "{backspace}good");
+
+    expect(input).toHaveValue("This is a good example");
+  });
+
+  test("emit click on button for {space}", () => {
+    const handleClick = jest.fn();
+    const { getByRole } = render(<button onClick={handleClick}>foo</button>);
+    const button = getByRole("button");
+
+    button.focus();
+    userEvent.type(button, "{space}", { skipClick: true });
+
+    expect(handleClick).toHaveBeenCalledTimes(1);
+  });
+});
```

Comment:
All other example use single quotes for `js` and double quotes in `jsx`.
Maybe we should lint the examples.